assignments/this.js

```diff
@@ -13,14 +25,55 @@
 
 // code example for Window Binding
 
+const info = {
+    'name': this.name
+};
+console.log(info.name)
+
+
 // Principle 2
 
 // code example for Implicit Binding
 
+const info = {
```

Comment:
this isn't quite right on implicit binding. Implicit binding is when you call a method on an object, then any `this` in that method are bound to that object